#!/usr/bin/env python
PACKAGE = "bezier_replanner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("step", int_t, 0, "step to set points", 5)
gen.add("resampling_num", int_t, 0, "resampling number", 5, 1)
gen.add("srcLane", str_t, 0, "src lane topic nmae", "/based/lane_waypoint_raw")
gen.add("dstLane", str_t, 0, "dst lane topic nmae", "/based/lane_waypoint_array")
gen.add("replanning_mode", bool_t, 0, "replanning mode", True)

exit(gen.generate(PACKAGE, "bezier_replanner", "bezierReplanner"))